### PR TITLE
CLI: Do not propagate fatal errors when constructing configuration

### PR DESCRIPTION
**Pull request by jakelang**
Depends: #128

### CODE REVIEW

File: chisel/src/main.rs [hunk 1]
```diff
@@ -97,7 +96,7 @@ impl ChiselContext {
                 let mut config_itr = config_clone.iter();
                 // Read modules while there are still modules left.
                 while let Some(module) = config_itr.next() {
-                    module_confs.push(ModuleContext::from_yaml(module)?);
+                    module_confs.push(ModuleContext::from_yaml(module).expect("**placeholder**"));
```
Comment (axic) [inline chisel/src/main.rs]: Placeholder?

**Comment by axic** [comment]
Rebased.

File: chisel/src/main.rs (hunk 1 above)
Comment (jakelang) [reply chisel/src/main.rs]: Fixed in the next PR.

Comment (axic) [reply chisel/src/main.rs]: Fixed it here with using `unwrap`.

**Pull request merged**